### PR TITLE
try to fix tracing

### PR DESCRIPTION
**Pull request by AlexTugarev**
follow-up to https://github.com/gitpod-io/gitpod/pull/9281

- [x] /werft with-observability=true
- [x] /werft with-helm=true
- [x] /werft with-clean-slate-deployment

```release-notes
NONE
```

### CODE REVIEW

File: components/gitpod-protocol/src/util/tracing.ts [hunk 1]
```diff
@@ -28,7 +28,8 @@ export namespace TraceContext {
         if (!!parentCtx?.span) {
             const ctx = parentCtx?.span?.context();
             if (ctx && !!ctx.toTraceId() && !!ctx.toSpanId()) {
-                options.references = [opentracing.followsFrom(ctx)];
+                // options.references = [opentracing.followsFrom(ctx)];
+                options.childOf = ctx;
```
Comment (AlexTugarev) [inline components/gitpod-protocol/src/util/tracing.ts:32]: @geropl, wdyt, should we remove these entirely? 
frankly, I don't understand the issue at hand, but given I started https://github.com/gitpod-io/gitpod/pull/9281, I'd like to find an end to this story. 
what option do we have? if we revert, we end up with messed up logs.

Comment (AlexTugarev) [reply components/gitpod-protocol/src/util/tracing.ts:32]: @geropl, is this still an issue?

Comment (geropl) [reply components/gitpod-protocol/src/util/tracing.ts:32]: No, I think we should leave it as-is. The problems we faced were mostly solved by Mad's attempts to find dangling spans that were not `finish()`ed. :+1:

**Comment by AlexTugarev** [comment]
Closing, because I'm out of ideas on this.

**Pull request closed without merging**